client/src/index.css

```diff
@@ -0,0 +1,13 @@
+body {
```

Comment:
do we need this file?

Reply:
I guess so, removing this file causes error

Reply:
ya cuz you are importing this file from index.js.
When there is a problem, try to find out why it's not working and make a judgement based on our use case.